src/Subscriptions/Endpoints/ListSubscriptions.php

```diff
@@ -111,6 +114,13 @@ public function registerRoute()
                             'columns',
                         ],
                     ],
+                    'status' => [
+                        'type' => 'string',
+                        'required' => false,
+                        'sanitize_callback' => 'sanitize_text_field',
+                        'validate_callback' => [$this, 'validateStatus'],
```

Comment:
Why not just make this an array type with enum like we did in the v3 REST [endpoint](https://github.com/impress-org/givewp/blob/7088af3ccf966fa62fa9661512e5b0d7e72ec89a/src/API/REST/V3/Routes/Subscriptions/Actions/GetSubscriptionCollectionParams.php#L46)?

Reply:
Oh, nice! Updated on 318f67f70347e3a8cad3a2b4e438099971c9db71